rmf_traffic/src/rmf_traffic/schedule/Mirror.cpp

```diff
@@ -274,20 +274,20 @@ Version Mirror::latest_version() const
 std::optional<ItineraryView> Mirror::get_itinerary(
   const std::size_t participant_id) const
 {
+  ItineraryView itinerary;
   const auto p = _pimpl->states.find(participant_id);
   if (p == _pimpl->states.end())
   {
     // If we don't have a state, it's possible that we have a description for
     // this participant but never received a state. In that case we should
     // return an empty itinerary, not a nullopt.
     if (_pimpl->participant_ids.count(participant_id) > 0)
-      return {};
+      return itinerary;
```

Comment:
Apologies for the nitpick but can we change this PR to simply

```
return ItineraryView{};
```

I prefer to make it explicit that an empty object is being returned. It won't be obvious at a glance that `itinerary` never gets filled up with anything, and there's a risk that a future change could slip some values into it when it's meant to be empty.

Reply:
Done https://github.com/open-rmf/rmf_traffic/pull/83/commits/2cb933bb6437b3c9f1ef9ef0258a70d9e47aaa35